staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -229,8 +229,6 @@ func init() {
 // To add a new feature, define a key for it above and add it here. The features will be
 // available throughout Kubernetes binaries.
 var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
-	AdmissionWebhookMatchConditions: {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
Why remove this?

Reply:
in rebasing, a double entry for this was created... mooted with subsequent rebasing